components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -36,6 +36,13 @@
             <bean class="org.wso2.carbon.apimgt.rest.api.util.exception.GlobalThrowableMapper" />
             <ref bean="cors-filter" />
         </jaxrs:providers>
+        <jaxrs:properties>
+            <!-- This is added to catch interceptor level exceptions in GlobalThrowableMapper. -->
+            <entry key="map.cxf.interceptor.fault" value="true" />
+            <!-- This is added to restrict the size of attachments sent through the requests. -->
+            <entry key="attachment-max-size" value="#{systemProperties['rest.api.store.attachment.max.size'] != null ?
+              systemProperties['rest.api.store.attachment.max.size'] :  T(java.lang.Integer).toString(10485760) }"/>
```

Comment:
shall we make it `rest.api.devportal.attachment.max.size`

Reply:
Fixed with 3bdfbda175ae8e9da3cd2ce8933709f3652b5778